rotkehlchen/data/nodes.json

```diff
@@ -8,48 +8,32 @@
         "blockchain": "ETH"
     },
     {
-        "name": "mycrypto",
-        "endpoint": "https://api.mycryptoapi.com/eth",
-        "weight": 0.15,
-        "owned": false,
-        "active": true,
-        "blockchain": "ETH"
-    },
-    {
-        "name": "blockscout",
-        "endpoint": "https://mainnet-nethermind.blockscout.com/",
-        "weight": 0.10,
+        "name": "cloudflare",
+        "endpoint": "https://cloudflare-eth.com/",
+        "weight": 0.20,
         "owned": false,
         "active": true,
         "blockchain": "ETH"
     },
     {
-        "name": "avado pool",
-        "endpoint": "https://mainnet.eth.cloud.ava.do/",
-        "weight": 0.05,
+        "name": "flashbots",
+        "endpoint": "https://rpc.flashbots.net/",
+        "weight": 0.20,
         "owned": false,
         "active": true,
         "blockchain": "ETH"
     },
     {
-        "name": "1inch",
-        "endpoint": "https://web3.1inch.exchange",
-        "weight": 0.15,
+        "name": "ankr",
+        "endpoint": "https://rpc.ankr.com/eth",
+        "weight": 0.20,
         "owned": false,
         "active": true,
         "blockchain": "ETH"
     },
     {
         "name": "myetherwallet",
-        "endpoint": "https://names.mewapi.io/rpc/eth",
-        "weight": 0.15,
-        "owned": false,
-        "active": true,
-        "blockchain": "ETH"
-    },
-    {
-        "name": "cloudflare",
-        "endpoint": "https://cloudflare-eth.com/",
+        "endpoint": "https://nodes.mewapi.io/rpc/eth",
         "weight": 0.10,
```

Comment:
What criteria did you use to rank the weight of the nodes? Did you perform any comparison?

Reply:
By "Load time" criteria on ethereumnodes.com
![image](https://user-images.githubusercontent.com/63492346/206468953-2ad88bb4-3726-446e-86fe-370e935a4193.png)